go.mod

```diff
@@ -3,107 +3,37 @@ module github.com/medibloc/panacea-core/v2
 go 1.15
 
 require (
```

Comment:
Well, in my idea, all the deleted dependencies are not needed.
I think if there are no problems, we can continue as it is.